roles/common/tasks/networking.yml

```diff
@@ -13,6 +13,16 @@
   with_items: network_interfaces_d
   tags: ['prereboot']
 
+- name: get uname -n
+  command: uname -n
+  register: uname
+  tags: etc_hosts
+
+- name: create 'uname -n' as fact 'uname_host'
+  set_fact:
+    uname_host: "{{ uname.stdout }}"
+  tags: etc_hosts
```

Comment:
Because of the naming conventions with dot notation - and the fact that I don't know what `platform.node()` might be on any given system (ref:  https://github.com/ansible/ansible/blob/5af1cda7c93375bc84296c641ace49bca8657e6c/lib/ansible/module_utils/facts.py#L163).  

You mentioned me `uname -n` was the reliable name - so I got it directly to avoid any inconsistencies.  If ansible_nodename is the go-to canonical, reliable device name then I'll definitely change this.